readyapi4j-groovy-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/RestRequestDslTest.groovy

```diff
@@ -255,6 +257,26 @@ class RestRequestDslTest {
         assert assertion.script == 'assert messageExchange.responseHeaders["Content-Type"].contains( "text/xml")'
     }
 
+    @Test
+    void addsExtractor() {
+        final String[] extractedProperty = { '' }
```

Comment:
The normal syntax is [ '' ], I think, although IMO new String[1] may be clearer.

Come to think of it, why are we using an array rather than List or just a String variable? Do we have to for some reason?